cms/envs/common.py

```diff
@@ -310,9 +317,22 @@
     OPENEDX_ROOT / 'core' / 'lib' / 'license' / 'templates',
     CMS_ROOT / 'djangoapps' / 'pipeline_js' / 'templates',
 ]
+MAKO_TEMPLATES['lms.main'] = lms.envs.common.MAIN_MAKO_TEMPLATES_BASE
+
+
+def _make_main_mako_templates(settings):
```

Comment:
At first glance, it looks like this and `_make_locale_paths()` are just copies of the implementations in `lms.envs.common`.  Given that we're already importing that module in this file, should we just reference these functions from there also?

Reply:
Yes - AFAIK there are no differences. I give it a go.